cmake/FindScala.cmake

```diff
@@ -0,0 +1,41 @@
+set(_SCALA_PATHS
```

Comment:
Did you write this file? Or did you find it somewhere online? If so, please put a comment with the link.

If you wrote it, double check existing solutions for inspiration